src/content/docs/en/guides/cms/hygraph.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Hygraph & Astro
+description: Add content to your Astro project using Hygraph as a CMS
+type: cms
+service: Hygraph
+i18nReady: true
+---
+import FileTree from '~/components/FileTree.astro'
+
+[Hygraph](https://hygraph.com/) is a federated content management platform. It uses a simple GraphQL endpoint for fast distribution of content.
```

Comment:
I've updated the language to be more neutral, and highlight the way the user fetches content.
```suggestion
[Hygraph](https://hygraph.com/) is a federated content management platform. It exposes a GraphQL endpoint for fetching content.
```